lib/fluent/plugin/out_fork.rb

```diff
@@ -62,9 +63,11 @@ def emit(tag, es, chain)
           end
         end
 
-        values.reject{ |value| value.to_s == '' }.each do |value|
+        values.reject{ |value| value.to_s == '' }.each_with_index do |value, i|
```

Comment:
In my opinion, I think it's ok because empty value will not be emitted by this plugin.
However what do you think about which is the most natural behavior?
I can fix this implementation if count also empty value is natural hehavior.